### ISSUE TITLE
Play nice with other dialers

### ISSUE THREAD

**Issue by tnyeanderson** [SimpleMobileTools/Simple-Contacts#395]
I use Simple Contacts as my default contacts app, and would love a Simple Dialer, but I know I will have to be patient for that one. I just found [Koler](https://github.com/Chooloo/call_manage), but when I try to dial out from Koler, it keeps forcing me to choose Simple Contacts as my default dialer. The same happens if I try to dial out from Emerald Dialer. Incoming calls work fine. If I uninstall Simple Contacts, the problem disappears and Koler is able to act as my dialer.

What could be causing this?

---

## Device Information
Model: LG V30
Android Version: 8.0.0 (Thanks, LG)
Simple Contacts Version: 6.3.1

## Steps to reproduce:
* Install Koler or Emerald and make it your default dialer
* Try to dial out by clicking a contact or typing a number

## Results:
"Complete action with" dialog shows up, offering the system app (Call Management) and Simple Contacts as options (Koler and Emerald don't show up here).

---

This could be a problem in the other apps, but since it happens with both, I figured I'd check here first.

**Comment by tibbi** [comment]
your other dialer apps might not be handling the required intent properly, nothing to be changed from by side there. The system determines what apps are capable of handling those intents.

**Comment by tnyeanderson** [comment]
For posterity and educational purposes: once I set Koler as my default dialer, I had to select Call Management in the "complete action with" dialog. That makes Koler work for phone calls. Not sure why it doesn't appear in that list, as I compared the Manifests for permissions/intents and Koler has all included here and then some. Maybe one day they'll figure it out.... hopefully after Simple Dialer is released ;)

Thanks so much for all of your important and difficult work. I have donated before and will continue to do so for this awesome suite of open source apps!
